typings/react-live.d.ts

```diff
@@ -15,6 +15,10 @@ export function LiveEditor(props: HTMLAttributes<HTMLElement>): JSX.Element
 export function LiveError(props: HTMLAttributes<HTMLElement>): JSX.Element
 export function LivePreview(props: HTMLAttributes<HTMLElement>): JSX.Element
 
-export class Editor extends Component<HTMLAttributes<HTMLElement>, {}>{}
+export type EditorProps = HTMLAttributes<HTMLElement> & {
```

Comment:
I'm not familiar with TypeScript at all, so let me know if I screwed this up somehow.